protobufs/stacks/signer/v1/requests.proto

```diff
@@ -16,7 +16,8 @@ message BitcoinTransactionSignRequest {
 
 // Represents a request to sign a Stacks transaction.
 message StacksTransactionSignRequest {
-  // The aggregate public key that will sign the transaction.
+  // This is the bitcoin aggregate key that was output from DKG. It is used
+  // to identifyy the signing set for the transaction.
```

Comment:
```suggestion
  // to identify the signing set for the transaction.
```

Reply:
Oh man, for some reason I thought I got all of these.